energy-integration/fees-collector/src/fees_accumulation.rs

```diff
@@ -25,14 +27,22 @@ pub trait FeesAccumulationModule:
             self.known_tokens().contains(&payment.token_identifier),
```

Comment:
Should have an `else` clause for payment being locked_token. Alternatively, add locked_token_id in `known_tokens` at init.

Reply:
I would not do an else check, as I already have the same check below. I've taken the assumption that the locked_token_id is already whitelisted, but I've also added it in the init function just for safety.

Reply:
Update. It does not matter if you set the locked_token_id in the init function, as add_known_tokens endpoint overwrites the storage anyway. So either way, the locked tokenId must be set when calling this endpoint.